docs/gettingstarted.rst

```diff
@@ -50,16 +50,14 @@ You can also easily deploy your own REANA cloud instance by using the
 
 .. code-block:: console
 
-   $ # install kubectl 1.14.0 and minikube 1.0.0
+   $ # install kubectl 1.16.3 and minikube 1.5.2
```

Comment:
If we are changing for minikube >= 1.5.2  then there are two more places to edit:

```
Makefile:96:    $(error "Please install Minikube v1.0.0 or higher")
docs/administratorguide.rst:177:version 1.0.0 is known to work the best).
```